projects/common/src/telemetry/providers/rudderstack/rudderstack-provider.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { Dictionary } from '../../../utilities/types/types';
+
+import { apiObject, identify, load, page, track } from 'rudder-sdk-js';
+import { TelemetryProviderConfig, UserTelemetryProvider, UserTraits } from '../../telemetry';
+
+export interface RudderStackConfig extends TelemetryProviderConfig {
+  writeKey: string;
```

Comment:
what is a `writeKey`? can we give it a better name?

Reply:
@anandtiwary Given we are creating an interface customised for RudderStack, it makes sense to use `WRITE_KEY` as the name here. Reference of RudderStack docs - https://www.rudderstack.com/docs/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/